lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Context/RootContextLauncher.cs

```diff
@@ -56,8 +56,17 @@ internal ContextRuntime GetRootContext()
         {
             if (_rootContext == null)
             {
-                // TODO[JIRA REEF-1167]: Remove use of this constructor.
-                _rootContext = new ContextRuntime(Id, _rootServiceInjector, _rootContextConfiguration);
+                // TODO[JIRA REEF-1167]: Remove use of deprecated ContextRuntime constructor and deprecatedContextConfiguration
+                var deprecatedContextConfiguration = _rootContextConfiguration as ContextConfiguration;
```

Comment:
`ContextConfiguration` is an implementation of `IConfiguration`, which is only there to hold the Context ID as of current. We are looking to remove `ContextConfiguration` from our codebase, but will still need to be supported for 0.14 release.